src/encoded/static/components/cart/cart.js

```diff
@@ -822,6 +824,66 @@ FileFacets.defaultProps = {
 };
 
 
+/**
+ * Display a button that links to experiment search results. This *only* works for Experiments, not
+ * FCCs for now.
+ */
+const MAX_SEARCHABLE_DATASETS = 125; // Maximum number of datasets for search-results links.
+const CartDatasetSearch = ({ elements }, reactContext) => {
+    const [isWarningVisible, setIsWarningVisible] = React.useState(false);
+
+    // * For now only include experiments. Might need to expand this in the future.
+    const experimentElements = elements.filter(element => element.includes('/experiments/'));
```

Comment:
Are the leading and trailing '/' needed?

Reply:
This eliminates the possibility of confusion if some other type with “experiments” in it comes up.